rfcs/0005-ftd-space.ftd

```diff
@@ -0,0 +1,112 @@
+-- import: fastn.com/rfcs/lib
+
+-- lib.rfc: RFC-5: Introducing `ftd.space` and `ftd.empty` Variables to `ftd`
+id: 0005-ftd-space
+status: pending
+
+The purpose of this RFC is to propose the introduction of two new variables,
+`ftd.space` and `ftd.empty`, to the `ftd` library. These variables will
+facilitate the insertion of space characters and empty strings.
+
+
+-- lib.motivation:
+
+The addition of `ftd.space` and `ftd.empty` variables simplifies the process of
+adding space character and empty space. By providing these predefined variables
+for common formatting needs, makes `ftd` becomes more user-friendly when working
+with spacing and empty content.
+
+The `ftd.empty` can be used as default value of string type variable. This
+variable can later on append other strings on some events.
+
+-- end: lib.motivation
+
+
+
+-- lib.detailed-design:
+
+`ftd` will now contains two predefined variables. These will be included in
+`default bag` in interpreter.
+
+-- ds.h2: Variables Definition
+
+-- ds.h3: `ftd.space`
+
+The `ftd.space` variable is defined to return a space character as a string.
+
+-- ds.h3: `ftd.empty`
+
+The `ftd.empty` variable is defined to return an empty string.
```

Comment:
Change line to: The `ftd.empty` variable contains empty string. Also change the `ftd.space`. Variables do not return things. Functions return things.